packages/editor/src/components/post-preview-button/index.js

```diff
@@ -99,6 +99,18 @@ function writeInterstitialMessage( targetDocument ) {
 	targetDocument.close();
 }
 
+/**
+ * A button for post preview. It uses WordPress data selectors to get the necessary data for the preview, such as the post ID, the current post link, the preview link, and whether the post is saveable and viewable. When the button is clicked, it opens a new window or tab for the preview, writes the interstitial message to this window, and then navigates to the actual preview link. The button is disabled if the post is not saveable.
```

Comment:
I think we should have a bit higher level description..

```suggestion
 * Renders a button that opens a new window or tab for the preview, writes the interstitial message to this window, and then navigates to the actual preview link. The button is not rendered if the post is not viewable and disabled if the post is not saveable.
```

Reply:
Also just noticed this.. We should probably wrap this comment in more lines to be consistent with our code base. I thought we had a linting rule for the max length of comments line, but it seems we don't 😄